clearpath_manipulators/config/planning/pipeline.yaml

```diff
@@ -1,2 +1,2 @@
-planning_pipelines: ['chomp', '_ompl', 'pilz']
-default_planning_pipeline: '_ompl'
+planning_pipelines: ['chomp', 'pilz', 'ompl']
+default_planning_pipeline: 'ompl'
```

Comment:
The MR title says "Rename `ompl` to `_ompl`" but this appears to be doing the opposite. Is the title wrong, or is this change wrong?

Reply:
Yeah, I got it backwards on the commit message